src/Compilers/VisualBasic/Portable/BoundTree/BoundObjectCreationExpression.vb

```diff
@@ -9,12 +9,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
     Friend Partial Class BoundObjectCreationExpression
 
-        Public Sub New(syntax As SyntaxNode, constructorOpt As MethodSymbol, arguments As ImmutableArray(Of BoundExpression), initializerOpt As BoundObjectInitializerExpressionBase, type As TypeSymbol, Optional hasErrors As Boolean = False)
-            Me.New(syntax, constructorOpt, Nothing, arguments, initializerOpt, type, hasErrors)
+        Public Sub New(syntax As SyntaxNode, constructorOpt As MethodSymbol, arguments As ImmutableArray(Of BoundExpression), initializerOpt As BoundObjectInitializerExpressionBase, type As TypeSymbol,
+                       Optional hasErrors As Boolean = False, Optional defaultArguments As BitVector = Nothing)
+            Me.New(syntax, constructorOpt, Nothing, arguments, defaultArguments:=defaultArguments, initializerOpt, type, hasErrors)
         End Sub
 
         Public Function Update(constructorOpt As MethodSymbol, arguments As ImmutableArray(Of BoundExpression), initializerOpt As BoundObjectInitializerExpressionBase, type As TypeSymbol) As BoundObjectCreationExpression
-            Return Update(constructorOpt, Nothing, arguments, initializerOpt, type)
+            Return Update(constructorOpt, Nothing, arguments, DefaultArguments, initializerOpt, type)
```

Comment:
>DefaultArguments [](start = 62, length = 16)

Can we pass it explicitly?

Reply:
I missed removing this function, will remove altogether.

---
In reply to: [190109952](https://github.com/dotnet/roslyn/pull/26781#discussion_r190109952) [](ancestors = 190109952)

Reply:
Scratch that, quick glance missed that this was preexisting. I'll fully-qualify this and the `Nothing` arg.

---
In reply to: [190380797](https://github.com/dotnet/roslyn/pull/26781#discussion_r190380797) [](ancestors = 190380797,190109952)